features/step_definitions/provider_steps.rb

```diff
@@ -39,6 +39,27 @@ def import_simple_layout(provider)
   Account.find_by_domain(domain).try!(&:destroy)
 end
 
+When "{provider} wants to log in" do |provider|
+  set_current_domain provider.external_admin_domain
+  visit provider_login_path
+end
+
+Given "{provider} tries to log in" do |provider|
+  username = provider.admins.first.username
+  set_current_domain provider.external_admin_domain
+  visit provider_login_path
+  fill_in 'Email or Username', with:  username
+  fill_in 'Password', with: "supersecret"
+  click_button 'Sign in'
+end
```

Comment:
If I'm not missing anything, this step is the same as https://github.com/3scale/porta/blob/3898a60f1ba8d6562a79364e215ac78ce798cea7/features/step_definitions/session_steps.rb#L3-L6